src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import axios from 'axios';
 import ReviewDisplay from './components/reviewdisplay';
 import Reviews from './components/searchreviewusingmovie';
+import Navbar from './components/navbar';
 
 const client = axios.create({
     baseURL: 'https://localhost:7035/api/',
```

Comment:
Lets break this out into a seperate folder called "Services" and within that folder of services we would have different API routes that we reach out to. This helps us to reuse the same code so we dont have to keep copy and pasting code over and over for the same request.